src/photos/ducks/upload/styles.styl

```diff
@@ -130,30 +130,6 @@ progress.upload-queue-progress::-moz-progress-bar
         font-weight    bold
         line-height    2rem
 
-    .item-loading
-        @extend $icon-16
-        @extend $icon-spinner-blue
-
-    .item-canceled
-        @extend $icon-16
-        @extend $icon
-        background-image embedurl('./assets/icons/icon-cross.svg')
```

Comment:
these SVG are still used somewhere else or can we remove them?

Reply:
Right! Apparently not used anymore. Deleted now! :)